### PR TITLE
396 simplify version numbers

### PR DESCRIPTION
**Pull request by aldipower**
Display 'Current'- or 'Out-of-date'-text instead of the timestamp in the version field of the task list for more readability.
According to #396 .
'Current' is displayed as .text-muted and 'Out-of-date' as .text-warning.

### CODE REVIEW

**Comment by aldipower** [comment]
Maybe 'Current' should be displayed as normal text and not as .text-muted.
Now the colors are a little bit bumpy in the horizontal line with Status and Updated.

**Comment by drexin** [comment]
Not sure about the version numbers, we use them for rollbacks. It should at least be possible to somehow get the real value.

**Comment by aldipower** [comment]
You can get the timestamp by mouseover the field by the title-attribute. Is this sufficent?

**Comment by drexin** [comment]
It should be copyable. Can we maybe add something like: hover over version -> caption = "Click to copy version number" -> version number gets copied to clipboard on click?

**Comment by drexin** [comment]
Oh and maybe the display could also change from "Current" or "Out-of-date" to the actual timestamp on hover.

**Comment by aldipower** [comment]
"Directly copying to the clipboard via javascript is not permitted by any modern browser for security reasons"
Yeah, that would be a solution, changing the text to the timestamp on hover. Will build an example.

**Comment by drexin** [comment]
Ok, thanks.

**Comment by mlunoe** [comment]
@drexin You can always see the older version numbers in configuration. Isn't that enough?

I can live with click interaction, I'll let @drexin by the jugde of whether it is frustrating that it reverts back whenever the list is repainted (e.g. when number of tasks are scaled up, or down).

File: src/main/resources/assets/js/components/TaskListItemComponent.jsx [hunk 1]
```diff
@@ -117,9 +134,13 @@ define([
...
           </td>
           <td className="text-right">
-            <time dateTime={task.get("version").toISOString()} title={task.get("version").toISOString()}>
-              {task.get("version").toLocaleString()}
-            </time>
+            <span
+              title={task.get("version").toISOString()}
+              className={versionClassSet}
+              onClick={this.handleVersionClick}>{
+                this.state.versionShowTimestamp ?
```
Comment (mlunoe) [inline src/main/resources/assets/js/components/TaskListItemComponent.jsx]: Can we get this up in a variable above?
```
var versionNode = outdated ? "Out-of-date" : "Current";
if (this.state.versionShowTimestamp) {
  versionNode = task.get("version").toISOString()
}
```
And then:
```
<span
    title={task.get("version").toISOString()}
    className={versionClassSet}
    onClick={this.handleVersionClick}>
  {versionNode}
</span>
```

**Comment by mlunoe** [comment]
Other than that, code looks good :+1:

File: src/main/resources/assets/js/components/TaskListItemComponent.jsx (hunk 1 above)
Comment (aldipower) [inline src/main/resources/assets/js/components/TaskListItemComponent.jsx]: I've talked with @drexin and removed the click interaction. It's enough to see the older versions in the configuration.
So the 'variable above' is obsolete, isn't it?

Comment (mlunoe) [inline src/main/resources/assets/js/components/TaskListItemComponent.jsx]: Yes it is :)

**Comment by drexin** [comment]
:100:

**Pull request merged**